tests/functional/lib/helpers.js

```diff
@@ -117,7 +116,6 @@ define([
       .end()
 
       .execute(function () {
-        /* global $ */
```

Comment:
Maybe try `//*eslint-env jquery */` here instead of adding it globally in .eslintrc?